src/components/Table/TableHead/FilterHeaderRow/FilterHeaderRow.jsx

```diff
@@ -231,13 +228,12 @@ class FilterHeaderRow extends Component {
           .map((c, i) => {
             const column = columns.find(item => c.columnId === item.id);
             const columnStateValue = this.state[column.id]; // eslint-disable-line
-            const filterColumnOptions = options => {
+            const filterColumnOptions = memoize(options => {
```

Comment:
I don't understand why memoize is used here. The memoized `filterColumnOptions` is created and called only once in the mapping loop of the `ordering` before it is created again. Removing the memoize results in the same amount of calls to the sort function.

Perhaps the Combobox should be its own component that we pass through the `React.memo` HOC on instead?

Reply:
This one I didn't actually add I just moved, perhaps it doesn't work the way the adder thought it would, I can look into

Reply:
you're right that the memoize isn't really working here, I just put a TODO to investigate this code in the future and reverted my change